scripts/templates/devtools.tpl.d.ts

```diff
@@ -2,10 +2,6 @@
 /// <reference types="webdriver"/>
 
 declare namespace WebDriver {
-    interface ClientOptions {
-        isDevTools: boolean;
```

Comment:
why have you deleted this one? It's devtools protocol specific, a way to identify which automation protocol is currently used, webdriver or devtools

Reply:
it is already defined in `webdriver.d.ts`